js/plugins/chroma/src/index.ts

```diff
@@ -51,6 +51,10 @@ const ChromaRetrieverOptionsSchema = CommonRetrieverOptionsSchema.extend({
 
 export const ChromaIndexerOptionsSchema = z.null().optional();
 
+type ChromaClientParams =
+  | NativeChromaClientParams
+  | (() => Promise<NativeChromaClientParams>);
```

Comment:
I changed the params field type, rather than adding a separate `requestHeaders` field to the plugin options like Ollama has. 

This is because ChromaClientParams already has extensive support for fetch options. https://github.com/timothycarambat/chroma/blob/3d399cd0ef89bb2ebb09b35fbad2dd5e46ac727f/clients/js/src/ChromaClient.ts#L39